lesson9/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Inheritance, Overriding and Overloading
+nav_order: 9
+has_children: true
+---
+
+# Lesson 9: Inheritance, Overriding and Overloading
+
+## Goals
+
+- Check-in
+- Recap homework and previous classes.
+- What is _Inheritance_?
+- What is _Method Overriding_?
+- What is _Method Overloading_?
+- Break-out
+- Regroup and check-out
+
+## Check-in
+
+## Recap homework and previous class.
+
+- What is Object Oriented Programming?
+- What is the difference between a class and an object?
+- What is a constructor?
+- What are the different parts of a method?
+- What is the difference between a static and non static method?
+- What are the different visibilities we have in Java?
```

Comment:
Do we plan to ask here but then clarify it during the course? I think properly understanding this without inheritance is not really possible :-D

Reply:
Yeah. I discussed the different visibilities with them in the Recap Method session. And I plan to ask this to refresh their memory. If they do not remember, then we will do it again.